drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/remote/reusable/ReusableRemoteWebDriverToDestroy.java

```diff
@@ -0,0 +1,38 @@
+package org.jboss.arquillian.drone.webdriver.factory.remote.reusable;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+/**
+ * @author <a href="mailto:mjobanek@redhat.com">Matous Jobanek</a>
+ */
+public class ReusableRemoteWebDriverToDestroy {
```

Comment:
Why do we need it if there is already `ReusableRemoteWebDriver`? Can't we solve it just by using it?